plugins/ConnectedLivingSpace/CLSAddon.cs

```diff
@@ -966,5 +955,24 @@ private void CrewTransfered(GameEvents.HostedFromToAction<ProtoCrewMember, Part>
                 Debug.LogException(ex);
             }
         }
+
+        private void UpdateHighlighting()

```

Comment:
This was added to support frame updates, but subsequent changes in KSP.0.90.0 allow for the use of HighlightType.

This could be removed.